synthtool/gcp/templates/node_library/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,15 +6,20 @@ about: Create a report to help us improve
 
 Thanks for stopping by to let us know something could be better!
 
-**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+**PLEASE READ**
 
-Please run down the following list and make sure you've tried the usual "quick fixes":
+1) Is this a client library issue or a product issue?
+This is the client library for {{metadata['product']}}. We will only be able to assist with issues that pertain to the behaviors of this library. If the issue you're experiencing is due to the behavior of the product itself, please visit [{{metadata['product']}} Support page]({{metadata['product-url']}}/docs/getting-support) to reach the most relevant engineers.
```

Comment:
Haha, you can see what's available here:
https://github.com/googleapis/google-cloud-node/blob/master/libraries.json

You probably want:
- `product` => `name_pretty`
- `product-url` => `product_documentation`

In some cases we also have a link to the product issue tracker via `issue_tracker`.  I would double check that value is available on a library by library basis before using it though.

Reply:
Ah, nice. I sent a PR (that this PR now depends on): https://github.com/googleapis/google-cloud-node/pull/3008